assignment_3/Marksheet/src/Marksheet.java

```diff
@@ -0,0 +1,148 @@
+import java.util.Scanner;
+
+
+public class Marksheet
+{
+	/**
+	 * 
+	 * @param grades of students
+	 * @return Average of grades
+	 */
+	public float Calculate_Average(int grades[])
+	{
+		float Average=0;
+		int sum=0;
+		for(int i=0;i<grades.length;i++)
+		{
+			sum=sum+grades[i];
+		}
+		
+		Average=(float) (sum/grades.length);
+		
+		return Average;
+		
+	}
+	
+	/**
+	 * 
+	 * @param grades of students
+	 * @return maximum of grades
+	 */
+	public int Calculate_Max(int grades[])
+	{
+		int Max=grades[0];
+		for(int i=0;i<grades.length;i++)
+		{
+			if(Max<grades[i])
+			{
+				Max=grades[i];
+			}
+		}
+		
+		return Max;
+	}
+	
+	/**
+	 * 
+	 * @param grades of students
+	 * @return minimum of grades
+	 */
+	public int Calculate_Min(int grades[])
+	{
+		int Min=grades[0];
```

Comment:
It will result in exception if there are 0 number of students. Always perform a size check in an array while accessing element with index